examples/tms-tiled/index.html

```diff
@@ -3,11 +3,19 @@
   <head>
     <meta charset="utf-8" />
     <link rel="stylesheet" href="style.css" />
+<<<<<<< HEAD
+    <link rel="stylesheet" href="../../lib/leaflet/leaflet.css" />
+  </head>
+  <body>
+    <div id="map"></div>
+    <script src="../../lib/leaflet/leaflet.js"></script>
+=======
     <link rel="stylesheet" href="http://cdn.leafletjs.com/leaflet/v0.7.7/leaflet.css" />
   </head>
   <body>
     <div id="map"></div>
     <script src="http://cdn.leafletjs.com/leaflet/v0.7.7/leaflet.js"></script>
+>>>>>>> master

```

Comment:
Oops, I missed pressing save on that one! I'm working on trying to get automated tests so can fix that soon as well. Thanks for noticing!